src/components/configuration/ConfigBar.tsx

```diff
@@ -135,11 +139,9 @@ function ConfigBar({
                 </Box>
             ) : (
                 <Box>
-                    <Tooltip title={"Logg inn"}>
-                        <IconButton color={"primary"} href={"/api/auth/login"}>
-                            <LoginIcon />
-                        </IconButton>
-                    </Tooltip>
+                    <Button startIcon={<LoginIcon />} href={"/api/auth/login"}>
```

Comment:
Good suggestion to write out "Logg inn" :+1:

Is it just me, or would `endIcon` be more natural?

Reply:
Yeah, I agree. Fixed in https://github.com/mathiazom/rezervo-web/pull/100/commits/bc65565f9f332aad12021cf0572ad789eefb55bc